src/ContactModels/include/BipedalLocomotionControllers/ContactModels/ContinuousContactModel.h

```diff
@@ -0,0 +1,141 @@
+
+    double m_length; /**< Length of the rectangular contact surface */
+    double m_width; /**< Width of the rectangular contact surface */
+
+    /**
+     * Evaluate the contact wrench given a specific contact model
+     */
+    void computeContactWrench() final;
+
+    /**
+     * Evaluate the Autonomous System Dynamics of the derivative of a specific contact model
+     */
+    void computeAutonomousDynamics() final;
+
+    /**
+     * Evaluate the Control Matrix of the derivative of a specific contact model
+     */
+    void computeControlMatrix() final;
+
+    /**
+     * Set the parameters cannot change
+     * @param state std::unordered_map containing the mutable parameters of the system. To compute the contact
+     * wrench between two system (i.e. a link of a robot and the environment) the following data has
+     * to be available. If not an std::runtime_error is thrown.
+     *   - \a length (double): length in meters associated to the model
+     *   - \a width (double): width in meters associated to the model
+     */
+    void setImmutableParameters(const std::unordered_map<std::string, std::any>& parameters) final;
+
+public:
+
+    /**
+     * Constructor. It instantiate the value of the mutable parameters and the immutable parameters.
+     * @param immutableParameters std::unordered_map containing the immutable parameters of the
+     * system. The list of the required parameters could be found in @ref
+     * BipedalLocomotionController::ContactModels::setImmutableparameters
+     * @param mutableParameters std::unordered_map containing the mutable parameters of the
+     * system. The list of the required parameters could be found in @ref
+     * BipedalLocomotionController::ContactModels::setMutableparameters
+     */
+    ContinuousContactModel(const std::unordered_map<std::string, std::any>& immutableParameters,
+                           const std::unordered_map<std::string, std::any>& mutableParameters);
+
+    /**
+     * Constructor. It instantiate the value of the mutable parameters and the immutable parameters.
+     * @param parameters std::unordered_map containing the immutable and mutable parameters of the
+     * system. The list of the required parameters could be found in @ref
+     * BipedalLocomotionController::ContactModels::setMutableparameters
+     */
+    ContinuousContactModel(const std::unordered_map<std::string, std::any>& parameters);
+
+
+    /**
+     * Set the internal state of the model.
+     * @param state std::unordered_map containing the state of the system. To compute the contact
+     * wrench between two system (i.e. a link of a robot and the environment) the following data has
+     * to be available. If not an std::runtime_error is thrown.
+     *   - \a frame_transform (iDynTree::Transform): transformation between the link and the inertial frame;
+     *   - \a null_force_transform (iDynTree::Transform): transformation corresponding to a null force;
```

Comment:
In theory, there could be infinite transforms for which you get a null force right?

Reply:
In theory yes because it depends on the velocity. I can update the documentation as 
```
     *   - \a null_force_transform (iDynTree::Transform): transformation corresponding to a null force when the velocity of the system is zero;
```

Reply:
> In theory yes because it depends on the velocity.

I don't think I understood your remark :thinking:. In my opinion, it is not clear how to choose that transform.

Reply:
For us is the position given by the planner. In general, I don't know